src/resources/ble_darwin.mm

```diff
@@ -0,0 +1,1297 @@
+// Copyright (C) 2018 Toitware ApS.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; version
+// 2.1 only.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// The license can be found in the file `LICENSE` in the top level
+// directory of this repository.
+
+// Note: This is host, we can ignore malloc errors
+
+#include "../top.h"
+
+#include "../objects.h"
+#include "../objects_inline.h"
+#include "../event_sources/ble_host.h"
+
+#undef BOOL
+
+#import <CoreBluetooth/CoreBluetooth.h>
```

Comment:
Generally, we try to have system includes earlier than local ones. (Assuming that "import" is like an include).
I don't remember exactly why anymore, but there were some issues that appeared otherwise.

Reply:
That said. It probably doesn't matter, and with the `BOOL` undef it's easier this way.

Reply:
Which is a good idea, but, there are issues with #defines if it is done in any other order.